### ISSUE TITLE
Layout is misaligned

### ISSUE THREAD

**Issue by saishton** [wchill/steamSummerMinigame#77]
The part of the layout containing the players in game/lane and the rolling list of ability uses is appearing too high up in the column - the player data is almost within the expanded box containing the DPS/Health/Active abilities whilst the rolling list of ability uses finishes approximately the same distance above the bottom of the 'speaker' on the TV.

~~I also don't appear to be getting a count of players in the current lane.~~
EDIT: After a refresh, I am now getting a lane count.

I've attached an image below:
![layoutbug](https://cloud.githubusercontent.com/assets/12876769/8148005/ba2f1c50-127e-11e5-970c-1b0930440612.png)

**Comment by wchill** [comment]
What is your screen resolution?

**Comment by saishton** [comment]
1280x800 - it also stays in that layout even if the window isn't maximised, if that makes any difference.

**Comment by crimzonrayne** [comment]
Couldn't reproduce at same resolution. Players in lane number missing in Chrome, X/Y players in room missing in FF, but no layout misalign.

**Comment by saishton** [comment]
I haven't really done and JS coding before, but having a glance through the code, we have this:

```javascript
function fixActiveCapacityUI(){
	$J('.tv_ui').css('background-image','url("'+GITHUB_BASE_URL+'game_frame_tv_fix.png")');
	$J('#activeinlanecontainer').css('height','134px');
	$J('#activitycontainer').css('height', '270px');
	$J('#activityscroll').css('height', '270px');
}

Adjusting the 134px to 154px in this pretty much fixes the issue (for me)

**Comment by welwood08** [comment]
The layout bug has been annoying me too.
@saishton Would you mind making that change into a pull request? A closed issue is unlikely to be seen by many people or acted upon by a contributor, a pull req is the right way to share the solution with everybody.

**Comment by saishton** [comment]
I assumed it was just affecting me - that's why I initially closed the issue. As its affecting other people, I'll reopen it - although I don't know why it only appears to be affecting certain people...

As for making the change, I'm not really sure how to do it - I don't really have any familiarity with GitHub, but the fix is above, so hopefully someone with a little more know-how can do it!

**Comment by welwood08** [comment]
Fix in PR #97

**Comment by saishton** [comment]
Thanks! I'll close this now then!
